cfme/web_ui/__init__.py

```diff
@@ -1957,7 +1957,8 @@ def click_path(self, *path, **kwargs):
         See :py:meth:`expand_path` for more informations about synopsis.
         """
         node = self.expand_path(*path, **kwargs)
-        sel.click(node)
+        if node is not None:
```

Comment:
Specific to test_rolename_required_error_validation, the registered fill method for the BootstrapTree ends up calling click_path() on an empty list, so `node` ends up being None.

This change would only mitigate the failure. I think I'd like to update `_fill_bstree_seq` to return None when the values list is empty. Prototyping now.